x-pack/plugins/security_solution/public/overview/components/detection_response/hooks/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export { useNavigateToTimeline } from './use_navigate_to_timeline';
```

Comment:
Getting an error in VSCode for this one, wondering if anyone has seen this one:
File  '.../hooks/use_navigate_to_timeline.tsx' is not listed within the file list of project '.../kibana/x-pack/plugins/security_solution/tsconfig.json'. Projects must list all files or use an 'include'. Not sure I get why this is a problem or how to resolve it. or if its just by vscode.